File: concept/15_2D_array/row_wise_sum/row_wise_sum.dart
void main() {
  List<List<int>> arr = [
    [3, 4, 11],
    [2, 12, 1],
    [7, 8, 7]
  ];

  for (var element in arr) {
    int sum = element.fold(0, (p, e) => p + e);
    print("$element => sum is $sum");
  }
}

File: concept/15_2D_array/row_wise_sum/row_wise_sum.cpp
#include <iostream>
using namespace std;

int main()
{
    int arr[3][3] = {{3, 4, 11}, {2, 12, 1}, {7, 8, 7}};

    for (int i = 0; i < 3; i++)
    {
        for (int j = 0; j < 3; j++)
        {
            cout << arr[i][j] << " ";
        }
        cout << endl;
    }

    for (int i = 0; i < 3; i++)
    {
        int sum = 0;
        for (int j = 0; j < 3; j++)
        {
            sum = sum + arr[i][j];
        }
        cout << "Row " << i + 1 << " sum is " << sum << endl;
    }
}